dashboards/ansible-tower/ansible-tower.json

```
@@ -0,0 +1,801 @@
+          {
+            "title": "Available Logtypes",
+            "layout": {
+              "column": 1,
+              "row": 13,
+              "width": 8,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.table"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Log SELECT count(*) FACET logtype "
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          },
+          {
+            "title": "Host Load - Last Week Comparision",
+            "layout": {
+              "column": 9,
+              "row": 13,
+              "width": 4,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.billboard"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Metric SELECT average(host.loadAverageOneMinute), percentile(host.loadAverageOneMinute) SINCE 7 days ago COMPARE WITH 1 week ago"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          },
+          {
+            "title": "Ansible Tower Log",
+            "layout": {
+              "column": 1,
+              "row": 16,
+              "width": 12,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "logger.log-table-widget"
+            },
+            "rawConfiguration": {
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Log SELECT *"
+                }
+              ]
+            }
+          }
+        ]
+      },
+      {
+        "name": "Infrastructure",
+        "description": null,
+        "widgets": [
+          {
+            "title": "CPU usage (%)",
+            "layout": {
+              "column": 1,
+              "row": 1,
+              "width": 4,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.line"
+            },
+            "rawConfiguration": {
+              "config": {
+                "bisect": {
+                  "position": null
+                }
+              },
+              "legend": {
+                "enabled": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": "3510613",
```

We probably want to set this to `0` so that all users will see the same experience.

```suggestion
                  "accountId": 0,
```

---

done.. updated the dashboard json